student.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :name, :classroom
+
   def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     super(age, name: name, parent_permission: parent_permission)
     @classroom = classroom
```

Comment:
- [ ] Please make sure when setting the classroom for a student it also adds it to the classrooms' students that serves as the 'belongs-to' side (a student belongs to a classroom).
  - You can consider using the below code.

```ruby
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```